xep-0384.xml

```diff
@@ -481,10 +481,10 @@
       <p>
         An OMEMO encrypted message is specified to include an &lt;encrypted&gt; element in the 'urn:xmpp:omemo:1' namespace. It always contains two child nodes, the &lt;header&gt; and the &payload; element.
         The &lt;header&gt; element has an attribute named 'sid' referencing the device id of the sending device and contains one or multiple &lt;keys&gt; elements, each with an attribute 'jid' of one of the recipients bare JIDs as well as one or multiple &lt;key&gt; elements.
-        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext that is the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
-        The ciphertext that is the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
+        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext containing the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
+        The ciphertext with the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
```

Comment:
I don't think this makes things clearer and it technically also isn't correct. The ciphertext doesn't _contain_ the encrypted element, the ciphertext _is_ the encrypted element. `that is` is the better choice in both cases IMO.

Reply:
OK, thanks.
That was also my first understanding of it.
The problem was that those two lines were too long and complex.
Therefore, I thought that the first change would be what was intended by the authors.
The new suggestion should simplify the text without changing the meaning of the sentences.

Reply:
Thanks, I like the new suggestions :+1: